<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertisseurTemp</class>
 <widget class="QMainWindow" name="ConvertisseurTemp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>403</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ConvertisseurTemp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lTitre">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Convertisseur de températures</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="70,70">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>40</number>
        </property>
        <property name="bottomMargin">
         <number>16</number>
        </property>
        <property name="horizontalSpacing">
         <number>7</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="2" column="0">
         <widget class="QLineEdit" name="eCelsius">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Entre 1 et 999</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lFahrenheit">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Degrés Fahrenheit</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lFahrenhei_convertisseur">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>         --.--</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lCelsius">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Degrés Celsius</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="bConvertir">
          <property name="text">
           <string>Convertir</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bEffacer">
          <property name="text">
           <string>Effacer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bQuitter">
          <property name="text">
           <string>Quitter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>486</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bQuitter</sender>
   <signal>clicked()</signal>
   <receiver>ConvertisseurTemp</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bEffacer</sender>
   <signal>clicked()</signal>
   <receiver>eCelsius</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bEffacer</sender>
   <signal>clicked()</signal>
   <receiver>lFahrenhei_convertisseur</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
